src/EditorFeatures/CSharpTest/Completion/CompletionProviders/KeywordCompletionProviderTests.cs

```diff
@@ -414,5 +414,117 @@ public async Task SuggestEventAfterReadonlyInStruct()
 ";
             await VerifyItemExistsAsync(markup, "event");
         }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.KeywordRecommending)]
+        [WorkItem(37223, "https://github.com/dotnet/roslyn/issues/37223")]
+        public async Task SuggestVarInForRefLoop0()
+        {
+            var markup =
+@"class C {
+    void F() {
+        for (ref $$
```

Comment:
teh right place for the tests is in RefKeywordRecommenderTests

that tests these fine-grained scenarios.  this test file is supposed to just be for validating that keywords show up :)

Reply:
Ah missed that good catch @CyrusNajmabadi.

Reply:
Thanks! I suspected that so few test cases and files here?! It turns out that they're in another project.

BTW, the correct place is `VarKeyword...` not `RefKeyword...`